source/unified-test-format/unified-test-format.rst

```diff
@@ -408,7 +439,13 @@ The structure of this object is as follows:
 
 .. _entity_client:
 
-- ``client``: Optional object. Defines a MongoClient object.
+- ``client``: Optional object. Defines a MongoClient object. In addition to
+  the configuration defined below, test runners MUST track the number of
+  connections checked out at any given time for the constructed MongoClient.
+  This can be done using a single counter and
+  `CMAP events <../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#events>`__.
+  Each ``ConnectionCheckedOutEvent`` should increment the counter and each
+  ``ConnectionCheckedInEvent`` should decrement it.
```

Comment:
I would propose putting the connection-tracking stuff in its own paragraph to preserve the one-line explanation for readability.

Should we clarify that this only applies to drivers that support connection pools? Additionally, I would assume it's only necessary for tests that include one or more `assertNumberConnectionsCheckedOut` operations.